tests/ci/stress_check.py

```diff
@@ -0,0 +1,275 @@
+    if os.path.exists(result_folder):
+        test_files = [f for f in os.listdir(result_folder) if os.path.isfile(os.path.join(result_folder, f))]
+        additional_files = [os.path.join(result_folder, f) for f in test_files]
+
+    if os.path.exists(server_log_path):
+        server_log_files = [f for f in os.listdir(server_log_path) if os.path.isfile(os.path.join(server_log_path, f))]
+        additional_files = additional_files + [os.path.join(server_log_path, f) for f in server_log_files]
+
+    additional_files.append(run_log_path)
+
+    status_path = os.path.join(result_folder, "check_status.tsv")
+    if not os.path.exists(status_path):
+        return "failure", "check_status.tsv doesn't exists", test_results, additional_files
+
+    logging.info("Found check_status.tsv")
+    with open(status_path, 'r', encoding='utf-8') as status_file:
+        status = list(csv.reader(status_file, delimiter='\t'))
+
+    if len(status) != 1 or len(status[0]) != 2:
+        return "error", "Invalid check_status.tsv", test_results, additional_files
+    state, description = status[0][0], status[0][1]
+
+    results_path = os.path.join(result_folder, "test_results.tsv")
+    with open(results_path, 'r', encoding='utf-8') as results_file:
+        test_results = list(csv.reader(results_file, delimiter='\t'))
+    if len(test_results) == 0:
+        raise Exception("Empty results")
+
+    return state, description, test_results, additional_files
+
+
+if __name__ == "__main__":
+    logging.basicConfig(level=logging.INFO)
+    temp_path = os.getenv("TEMP_PATH", os.path.abspath("."))
+    repo_path = os.getenv("REPO_COPY", os.path.abspath("../../"))
+    reports_path = os.getenv("REPORTS_PATH", "./reports")
+
+    check_name = sys.argv[1]
+    build_number = int(sys.argv[2])
+
+    if not os.path.exists(temp_path):
+        os.makedirs(temp_path)
+
+    with open(os.getenv('GITHUB_EVENT_PATH'), 'r', encoding='utf-8') as event_file:
+        event = json.load(event_file)
+
+    pr_info = PRInfo(event)
+
+    gh = Github(get_best_robot_token())
+
+    for root, _, files in os.walk(reports_path):
+        for f in files:
+            if f == 'changed_images.json':
+                images_path = os.path.join(root, 'changed_images.json')
+                break
+
+    image_name = "clickhouse/stress-test"
+
+    docker_image = image_name
+    if images_path and os.path.exists(images_path):
+        logging.info("Images file exists")
+        with open(images_path, 'r', encoding='utf-8') as images_fd:
+            images = json.load(images_fd)
+            logging.info("Got images %s", images)
+            if image_name in images:
+                docker_image += ':' + images[image_name]
+    else:
+        logging.info("Images file not found")
+
+    for i in range(10):
+        try:
+            logging.info("Pulling image %s", docker_image)
+            subprocess.check_output(f"docker pull {docker_image}", stderr=subprocess.STDOUT, shell=True)
```

Comment:
@alesapin this will hide the error, like [here](https://github.com/ClickHouse/ClickHouse/runs/4169444589?check_suite_focus=true), since `subprocess.check_output` returns output.

And some of scripts under `tests/ci` does not redirects `stderr` to `stdout` for `docker pull`.

Maybe at least log the last error:

```patch
diff --git a/tests/ci/stress_check.py b/tests/ci/stress_check.py
index a773028502..ea4776c24d 100644
--- a/tests/ci/stress_check.py
+++ b/tests/ci/stress_check.py
@@ -222,16 +222,17 @@ if __name__ == "__main__":
     else:
         logging.info("Images file not found")
 
+    last_error = None
     for i in range(10):
         try:
             logging.info("Pulling image %s", docker_image)
-            subprocess.check_output(f"docker pull {docker_image}", stderr=subprocess.STDOUT, shell=True)
+            last_error = subprocess.check_output(f"docker pull {docker_image}", stderr=subprocess.STDOUT, shell=True)
             break
         except Exception as ex:
             time.sleep(i * 3)
             logging.info("Got execption pulling docker %s", ex)
     else:
-        raise Exception(f"Cannot pull dockerhub for image docker pull {docker_image}")
+        raise Exception(f"Cannot pull dockerhub for image docker pull {docker_image}: {last_error}")
 
     build_config = get_build_config(build_number, repo_path)
     build_config_str = build_config_to_string(build_config)
```

But I guss it will be even better to remove this redirection altogether, and not only in `stress_check.py`:

```sh
$ gg subprocess.*docker.*pull.*stderr= tests/ci
tests/ci/ast_fuzzer_check.py:            subprocess.check_output(f"docker pull {docker_image}", stderr=subprocess.STDOUT, shell=True)
tests/ci/build_check.py:            subprocess.check_output(f"docker pull {image_name}:{image_version}", stderr=subprocess.STDOUT, shell=True)
tests/ci/functional_test_check.py:            subprocess.check_output(f"docker pull {docker_image}", stderr=subprocess.STDOUT, shell=True)
tests/ci/integration_test_check.py:                subprocess.check_output(f"docker pull {docker_image}", stderr=subprocess.STDOUT, shell=True)
tests/ci/stress_check.py:            last_error = subprocess.check_output(f"docker pull {docker_image}", stderr=subprocess.STDOUT, shell=True)
tests/ci/unit_tests_check.py:            subprocess.check_output(f"docker pull {docker_image}", stderr=subprocess.STDOUT, shell=True)
```

Thoughts?